st2common/st2common/util/virtualenvs.py

```diff
@@ -199,14 +199,14 @@ def create_virtualenv(virtualenv_path, logger=None, include_pip=True, include_se
     logger.debug('Running command "%s" to create virtualenv.', ' '.join(cmd))
 
     try:
-        exit_code, _, stderr = run_command(cmd=cmd)
+        exit_code, stdout, stderr = run_command(cmd=cmd)
     except OSError as e:
         raise Exception('Error executing command %s. %s.' % (' '.join(cmd),
                                                              six.text_type(e)))
 
     if exit_code != 0:
-        raise Exception('Failed to create virtualenv in "%s": %s' %
-                        (virtualenv_path, stderr))
+        raise Exception('Failed to create virtualenv in "%s":\n%s\n|||||\n%s' %
```

Comment:
I assume that is only temporary while you debug the Travis Bionic environment.

If this is meant to be permanent, then I am not a huge fan of this output syntax - the format itself isn't very readable (I recommend using `.format()`), and it doesn't produce very readable output.

Reply:
yes. This is just for extremely verbose debugging. :)

Reply:
:+1: Resolving this. I'll have another look once this PR is finished.